src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public Excavator(String machine) {
+        super(machine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(machine + " started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(machine + " stopped working.");

```

Comment:
The stop message is also not informative enough. It should clearly indicate that an `Excavator` has stopped working. For example: "Excavator stopped working."